### ISSUE TITLE
Unable to launch

### ISSUE THREAD

**Issue by rcrx** [vkbo/novelWriter#44]
Received the following error:

```
>>> ~/apps/novelWriter $ ./novelWriter.py 
qt5ct: using qt5ct plugin

(python3:19846): dbind-WARNING **: 13:26:22.325: Couldn't register with accessibility bus: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.
Traceback (most recent call last):
  File "./novelWriter.py", line 45, in <module>
    nw.main(inArgs)
  File "$PATH/apps/novelWriter/nw/__init__.py", line 197, in main
    nwGUI = GuiMain()
  File "$PATH/apps/novelWriter/nw/gui/winmain.py", line 66, in __init__
    self.docEditor  = GuiDocEditor(self, self.theProject)
  File "$PATH/apps/novelWriter/nw/gui/doceditor.py", line 94, in __init__
    self.initEditor()
  File "$PATH/apps/novelWriter/nw/gui/doceditor.py", line 147, in initEditor
    theOpt.setTabStopDistance(self.mainConf.tabWidth)
AttributeError: 'QTextOption' object has no attribute 'setTabStopDistance'
```

**Comment by vkbo** [comment]
Thanks for the report. That is most likely due to an older version of PyQt5 and/or Qt5. 

I will have a look at this on the weekend (hopefully) and add the needed checks to catch this.

**Comment by rcrx** [comment]
Sounds great. If it helps, here's some package information (distro is Linux Mint XFCE 19.2):

```
>>> ~/apps/novelWriter $ apt list --installed *qt5*
Listing... Done
libqt5core5a/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5dbus5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5designer5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
libqt5gui5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5help5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
libqt5network5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5positioning5/bionic,now 5.9.5+dfsg-0ubuntu2 amd64 [installed,automatic]
libqt5printsupport5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5qml5/bionic-updates,now 5.9.5-0ubuntu1.1 amd64 [installed,automatic]
libqt5quick5/bionic-updates,now 5.9.5-0ubuntu1.1 amd64 [installed,automatic]
libqt5sensors5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
libqt5sql5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed,automatic]
libqt5svg5/bionic,now 5.9.5-0ubuntu1 amd64 [installed]
libqt5test5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed,automatic]
libqt5webchannel5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
libqt5webkit5/bionic,now 5.212.0~alpha2-7ubuntu1 amd64 [installed,automatic]
libqt5widgets5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
libqt5x11extras5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
libqt5xml5/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed,automatic]
python-pyqt5/bionic,now 5.10.1+dfsg-1ubuntu2 amd64 [installed,automatic]
python-pyqt5.qtsvg/bionic,now 5.10.1+dfsg-1ubuntu2 amd64 [installed,automatic]
python-pyqt5.qtwebkit/bionic,now 5.10.1+dfsg-1ubuntu2 amd64 [installed,automatic]
python3-pyqt5/bionic,now 5.10.1+dfsg-1ubuntu2 amd64 [installed]
python3-pyqt5.qtsvg/bionic,now 5.10.1+dfsg-1ubuntu2 amd64 [installed]
qt5-gtk-platformtheme/bionic-updates,now 5.9.5+dfsg-0ubuntu2.3 amd64 [installed]
qt5-image-formats-plugins/bionic,now 5.9.5-0ubuntu1 amd64 [installed]
qt5-style-plugins/bionic,now 5.0.0+git23.g335dbec-2build5 amd64 [installed]
qt5ct/bionic,now 0.34-1build2 amd64 [installed]
```

**Comment by vkbo** [comment]
Just checked the documentation, and novelWriter requires at least Qt 5.10. Looks like you have 5.9 installed. If 5.9 is default on many systems, I will try to add some additional checks around features that are non-essential but need higher version, like the ability to set tab stop distance.

**Comment by vkbo** [comment]
Added a fix in PR #45. The particular feature that failed in your case is now disabled if running with Qt < 5.10. I tested in a virtual machine with version 5.5.1, and it seems to run fine.
